hiro/windows/utility.cpp

```diff
@@ -7,12 +7,6 @@ static constexpr u32 Windows7     = 0x0601;
 
 static auto Button_CustomDraw(HWND, PAINTSTRUCT&, bool, bool, bool, u32, const Font&, const image&, Orientation, const string&) -> void;
 
-static auto OsVersion() -> u32 {
-  OSVERSIONINFO versionInfo{0};
-  versionInfo.dwOSVersionInfoSize = sizeof(OSVERSIONINFO);
-  GetVersionEx(&versionInfo);

```

Comment:
The call to GetVersionEx was causing deprecation warnings. I starting looking into how to silence those when I realized this code is entirely unused, so I just removed it.